pom.xml

```diff
@@ -135,8 +135,26 @@
           <goals>deploy</goals>
         </configuration>
       </plugin>
+      <!-- TODO: remove when library parent POM enables FindBugs by default -->
+      <plugin>
+        <groupId>org.codehaus.mojo</groupId>
+        <artifactId>findbugs-maven-plugin</artifactId>
+        <configuration>
```

Comment:
Should you add a version to the plugin?

    [WARNING] 
    [WARNING] Some problems were encountered while building the effective model for org.jenkins-ci.tests:plugins-compat-tester:jar:0.0.3-SNAPSHOT
    [WARNING] 'build.plugins.plugin.version' for org.codehaus.mojo:findbugs-maven-plugin is missing. @ org.jenkins-ci.tests:plugins-compat-tester-aggregator:0.0.3-SNAPSHOT, /Users/dnusbaum/Projects/plugin-compat-tester/pom.xml, line 139, column 15